### PR TITLE
Head of Security's armour nerf

### PR DESCRIPTION
**Pull request by Kierany9**
Because nobody wants 2/3rds damage reduction on a goddamn coat. Now has the exact same protection as standard sec armour, except it's full-body.

Also buffs the bullet resistance on standard security armour from 15 to 30 as it hardly made a difference.

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
Perhaps both headgear and armor should do 30% on bullet damage instead?

**Comment by trifornt-tinfornt** [comment]
Still a tid too powerful in my opinion (HoS cap shouldn't have 60 melee resistance even now) but it works.

**Comment by Crystalwarrior** [comment]
60% melee resistance is the same as normal sec armor except full body. I see what you mean but I'd rather still have HoS armor as a very viable counter to melee.
This will incentivise HoS players to use riot gear more against pure melee tho

**Comment by meteoses** [comment]
Tase+cuff counters the armor, this isn't like the RTP armor where it just let you escape anything via it's bullshit RNG. The HoS armor doesn't protect you from misplays and doesn't make you immune/resistant to handcuff.

**Comment by Spacedong** [comment]
i think this is the most ironic thing i've seen meteoses say, ever. taser and handcuffs, holy shit ahahhaa

**Comment by Crystalwarrior** [comment]
You realise that taze+cuff only helps you if HoS is alone and completely by himself with no help arriving for a good, long while, right?
Chances are if you DO taze and cuff the HoS you still have to fullstrip them just to deal any semblance of damage, and that means making yourself completely exposed for a decent amount of time.
Not to mention that HoS is supposed to be a team player, and if HoS has at least one guy on his side you're almost certainly going to have to toss the HoS into space, forget the gear and switch your identity quick.

Also HoS armor DOES protect you from quite a lot of misplays. It's a very good counter against brute AND lasers, 75% melee resistance on FULL BODY and 80% melee resistance for the HoS beret, as well as 50% laser resistance on everything means that you have an amazing protection against TWO OF THE MOST PROMINENT DAMAGE TYPES IN THE GAME.

Syringe guns shouldn't be the only way of dealing with HoS.

**Comment by Spacedong** [comment]
yeah he knows that he's just trying to come up with an argument so it won't be nerfed :^) 
HOS armor is bullshit and everyone knows it

**Comment by meteoses** [comment]
Hun, for a good long while? You just tase, cuff, pull off the headset then it's over. The "But a lot of the time players have help" argument could apply to any piece of armor.

The stats you listed aren't the actual stats, you can literally see those with 4 mouse clicks, I suggest you do so.

**Comment by Spacedong** [comment]
yes its so easy to tase cuff and pull off the headset with your playstyle :^) why do you want one counter to everything? you're avoiding the fact that he's fucking invincible, he can take 10 fire axe hits to the chest, WIELDED, its horrible game design

**Comment by Crystalwarrior** [comment]
Hunnyhonman, we had a long discussion about this in coder chat. You're only telling us one specific "hard counter" that really isn't as reliable as you make it out to be, not to mention the stun reduction HoS grandcoat has.
HoS armor has way too many positives that don't come with any negatives. You get full-body melee, laser AND stun protection, not to mention we're not even bringing chemicals into equasion that make stuns a complete joke *cough*meth*cough*

**Comment by Spacedong** [comment]
not to mention that riot shields are laughably overpowered in blocking ranged and syringe guns too

**Comment by Crystalwarrior** [comment]
Overall the issue here is not JUST the HoS grandcoat, but the stupid combinations the HoS grandcoat can be used WITH.

**Comment by Spacedong** [comment]
the HOS was never intended to be some hitler-esque invincible traitor hunter-killer, he is supposed to be an above average officer who keeps others in line. however this intention was lost to the idiots over at /tg/

**Comment by meteoses** [comment]
The HoS isn't invincible, the HoS greatcoat is strong but that's kind of the point since the HoS is supposed to be the most defensively and offensively powerful role on the station; you can still easily just stun/slip a player who has it. The reality is that the players who you are targeting with this are just far more skilled than you, the armor is a scapegoat. Surely you've seen that over 90% of HoS players are a joke and get eaten alive with the armor on.

The RTP armor made you nearly invincible, the HoS armor makes you resistant to extremely bad players who don't understand the concept of stuns+cuffs>all, but won't save you from a good player. Meth, chemical implants, RTP armor, these things all save you if you make a shitty play, so they got nerfed; the HoS armor is just armor with good stats instead of some gamebreaking "here's a free second chance to win your fight" mechanic like those other things.

**Comment by Pyko1** [comment]
Personally I think the hos stuff is fine, its still stun/cuff and dead.

It's been same for ages and never seen it as an issue.

**Comment by Crystalwarrior** [comment]
I really don't like how HoS grandcoat leaves so little options to combat HoS, though. Melee and lasers are out of the question, stuns are reduced in effectiveness and it's full-body so targeting won't work around the issue, so what do you do then? Run, stun and cuff is incredibly risky and unreliable, not to mention the possibility of chemicals at play, but let's consider "vanilla HoS" for simplicity's sake.
HoS still gets a riot shield which blocks more ranged attacks, making both melee AND ranged very risky and less than viable against him. Targeting head is actually WORSE for melee since the beret has 80% melee reduction rate (which also makes barely any sense, all things considering. The only reason I can find for this is to reduce knockout chance on HoS but even then it's an incredibly stupid solution imho)
The only decent melee-based counter I can think of to HoS armor is armor-piercing weapons, and we don't really have any good armor piercing melee weapons short of spiked bat anyway - which doesn't really have that good of a "base damage" anyway. If you have a chainsaw you're better off just chipping away at their health with less than 10 damage as opposed to 50.

**Comment by meteoses** [comment]
Crystalwarrior, nothing you said counters the reality that the armor is just one stun away from not mattering, like all armor except for the RTP. You basically spent half of that argument hedging that somehow stun+cuff isn't a hard counter, it simply is. Most of your complaints are about other items. The risk in using stun+cuffs you attribute is your perception from a huge skill gap, same as every game where the players of lower ranks see a top player do something they couldn't personally pull off then they assume what was done must have been OP-needs-nerf.

**Comment by Crystalwarrior** [comment]
What? I actually said Stun+Cuff IS a hard counter and I'm also saying why it's not a good thing.

By the way, as a "top player" you should also be aware that in any multiplayer game, changes to game balance are bound to happen. If you can't deal with the fact that something you enjoy using but that certain something also causes a ton of balancing issues later on (bad/toofuckingood item synergy is a legit balancing concern for your information) then I can't really consider you a "top player".
lern2adaptskrub

**Comment by meteoses** [comment]
Right, but Crystal, this doesn't actually give a huge edge like you make it seem; the other things that have been nerfed all give you a free second chance to win the fight despite shit play; if you get stunned/slipped/poisoned/etc then it's over for you with the armor on.

This armor has been around for a full year and no one started complaining until they realized they might be able to get nerfs. It has been fine the whole time, these complaints only showed up in the last few months.

You should be using Stun/Slip+Cuff if you want to compete against skilled players, of course if you use a terrible strategy (for the situation) against a skilled player it will likely fail.

You already nerfed the overpowered functions, this is just armor with good values, values which you keep making up, the practical effectiveness that you just keep lying about.

"lern2adaptskrub" funny that your entire argument is that you are too poor a player to adapt to something that you've had over a year to adapt to; really ironic that you would state advice that you personally failed to take.

**Comment by Crystalwarrior** [comment]
stop assuming people are terrible at the game simply because they're suggesting a nerf lol
I'll stop humoring you if all of your arguments are going to simply boil down to "gitgudfagt"

**Comment by Kierany9** [comment]
I'll just chip in with the reason why I made this PR.

Decentralizing the meta is always good. Having a very limited pool of hard counters to everything and no soft counters **stagnates gameplay fast** and generally makes situations far less aproachable and unforgiving even for experienced players, let alone new players. Let me explain. If I stun a HoS as an antag, it's usually because they fucked up. I now have a single option which doesn't work if I'm in a public place or the HoS has backup, which is to handcuff them(revolvers/lasers will work but you should not be carrying these on your person as they are easily beaten by stuns and if you get caught with them you'll most likely be executed, brigged or at the very least have your equipment confiscated). If I don't have handcuffs then the HoS basically got a free escape from a situation anybody without the armour would be fucked in, I can only prolong the inevitable for so long with my limited stuns. Remember, he made the mistake of getting stunned, the armour prevents me from punishing it properly.

If I have a weapon, I can hit them with it, which reduces their stamina because they took damage and then reusing whatever stun I had and repeating until they are dead. However, if I didn't bring a decent weapon or if I had no spare stuns left, the HoS will be able to get up before I can properly kill him. Now the fight is even which is fair because we both made a mistake. This does not happen because the armour negates 65% of all incoming melee damage and thus, even the best weapons still take ridiculously long to even put a dent in their HP with no drawbacks. The HoS is probably packing some spare weapons/stuns(if he doesn't then he's an idiot for even trying to follow you and should be dead due to shitty play anyway, but once again the armour saves him) so you both have multiple options.

As the traitor, do you run? Do you try and bait the HoS into waisting his stuns? Do you try and throw shit at him? Or do you try and go ham and crit him because you know he's super low? If you got his shield then that puts you at a big advantage, do you be more agressive with it or do you not risk it? Do you have security radio access, if so, do you know if backup is coming? As the HoS, do you run? Do you stand your ground? How much ammo do you have? What weapons does he have? Do you know if you're going to receive help or not? Can you get to medbay quickly from here, or do you have healing on hand? These are questions that aren't defined by a meta because it depends on circumstance and overall makes for a more dynamic experience.

And finally, a question for meteoses, why does losing 15% melee resistance affect you so much if it supposedly doesn't help you at all in your own words? And keep in mind that 15% it lost was added to bullet resistance, which lets you survive an extra revolver shot, a second volley of C20 bullets, one/two more Glock/Sketchink shots and a volley from the AK/C90 doesn't leave you on death's door. Just as I see that the melee resistance is too strong and restricts options too heavily, the bullet resistance is too weak and does very little to protect you from them.

**TL;DR:** HoS armour *can* protect against shitty play, reducing the armour values simply makes things more even.

**Comment by meteoses** [comment]
Basically the answer to your hypothetical is that you should have just had cuffs/wires on you so that he'd die when he went down. The HoS is supposed to have an advantage over the assistant in the scenario you have created, so yeah, that outcome seems fair, things should fall in the HoS's favor more often than the guy with only a grey jumpsuit if both sides play a fight terribly.

I'm actually fine with the change, I don't really think it's needed though, the armor has been fine this whole time, no one complained until a month ago. This isn't a full nerf, more of an actual balancing and it makes the armor more versatile; which will probably just end up making it even more powerful since melee is the pinnacle of "I've totally already lost so I'll just try this". I just don't think it needs any changes since it's stats don't give you a second chance against a skilled player(which is how I feel things should be balanced, at a high skill level), since players who are serious about killing you will have cuffs instead of just wailing on you with a bat. If this makes people stop begging for nerfs then I guess I'll thank you for it, Kierany .

**Comment by Kierany9** [comment]
If the HoS got stunned by the assistant then he deserves to be at a disadvantage in the upcoming fight even if the assistant doesn't have cuffs, you shouldn't have to lose or flee from a fight that you managed to get a huge advantage in just because "oops i didnt make cuffs". If you didn't bring any sort of weaponry either then yes, that's your fault, but melee is the most common form(and AP melee is rare), and making it entirely useless because "lol hos armour" is just dumb. This nerf doesn't make it useless, it just means that you take 2x less damage rather than 3x less and turning a suicidal fight into a fair fight. It takes a whopping four less fire axe or esword hits to crit someone in HoS armour. This is fine.

And let's not forget the invincible juggernaut that is a traitor who stole the HoS's shit.

**Comment by meteoses** [comment]
If the HoS gets stunned he should die, but only if you have cuffs, that's pretty much the game, if you don't have cuffs then your enemy has a chance to defeat you. I kind of just think your example is one where "this player is so bad he didn't even have cuffs on him" rather than "the HoS is so OP that he survived". I actually thought you guys were straight nerfing the armor because everyone kept tossing around exaggerated numbers, but looking at it you are pretty much just straight up buffing the armor; nerfing it against by far the weakest of all damage types and buffing it against the strongest; ergo I approve of this change.

Invincible until a single stun lands...

**Comment by Kierany9** [comment]
> If the HoS gets stunned he should die, but only if you have cuffs, that's pretty much the game, if you don't have cuffs then your enemy has a chance to defeat you.

And I'm making those chances less ridiculously one-sided.

**Comment by Spacedong** [comment]
I love how you're juggling both the meth dental implant issue and the HOS armor issue. MY issue is that you only exist on this site to be concerned about issues that stop your powergaming ego fest. You've done nothing to contribute to the game at all. You can't cuff the HOS when he has backup and implants. The HOS can survive 10 fire axes to the chest and can survive being blown up provided he has internals on and isn't at the center of an explosion. Even a revolver or esword cannot go up against his ridiculous armor combination combined with a riot shield.  Cuffs isn't the only solution to everything like you make it out to be. Ohh hurr durr cuffs is the only hard counter. give me a fucking break

**Comment by meteoses** [comment]
Well the chances are only one sided because one side was so terrible at the game that they didn't even spend the 5 seconds it takes to make cuffs.

Ok spacedong, as I said, I thought it was a nerf, now I see it's a buff, so I approve of this change; you approve of this change because you still think it's a nerf, I approve of your ignorance, hun.

**Comment by Spacedong** [comment]
see this is the fucking problem with you, you have this air of superiority and condescending tone where nobody can take you seriously, cuffs shouldn't be the only counter to the HOS armor, there should be multiple counters, his armor shouldn't be deathsquad tier

**Comment by meteoses** [comment]
There are plenty of counters, stun+cuff is just the most common. Oh, maybe you didn't know, but there are only enough materials for like 2-5 thousand cuffs on the station, a very rare item indeed. Bombs, chemicals, lube/water, any of these will counter the armor as well if you follow up with cuffs. If you want some sort of non cuff counter, then I'm not sure what to say, buy a 357 or shoot the HoS with cyanide?

There are a ton of counters, I'm just not sure if you are dishonest or really just that bad, I mean can you blame me for talking down to you guys when you act like your only option to take down a HoS is to just go at them with melee; how about a little creativity of thought when you attack?

**Comment by Spacedong** [comment]
except the HOS armor blocks syringe guns along with his hat and riot shield.

"There are a ton of counters, I'm just not sure if you are dishonest or really just that bad"

now you personally attack me and say I'm shit for wanting to find an issue to this overpowered as fuck armor? you're a joke. HOS armor shouldn't be deathsquad tier and require elaborate counters, combined with implants he is invincible as well.

**Comment by meteoses** [comment]
What. No it doesn't block syringes. Please go into a private server and test this out for yourself, the armor doesn't do anything like that, it's not a handsuit, hun.

I guess the fact you think the HoS armor blocks syringes goes to show that you definitely aren't "dishonest" and most definitely "really just that bad"; begging for nerfs for an item you don't even understand LOL. Yeah, I agree, the HoS armor that blocks syringes actually is OP, please nerf it for us Spacedong, please help us.

**Comment by Limed00d** [comment]
Yeah sure.

**Comment by Lawligagger** [comment]
sure http://i.imgur.com/xHWg2zL.png

**Comment by Crystalwarrior** [comment]
Two heads approved, why no merge??

**Comment by Carbonhell** [comment]
Iunno, you're a member too, why don't you?Merging

**Comment by Carbonhell** [comment]
HOL UP changelog is bugged

File: html/changelogs/Kierany9 - HoS Armour Balance [hunk 1]
```diff
@@ -0,0 +1,37 @@
+################################
```
Comment (Carbonhell) [inline html/changelogs/Kierany9 - HoS Armour Balance]: file lacks .yml extension

**Comment by Kierany9** [comment]
guys i fixed the logs you can merge now

**Comment by Pyko1** [comment]
approved and the codes issues seem to have been fixed, merging.
